.github/workflows/ci.yaml

```diff
@@ -39,29 +39,22 @@ jobs:
 
       - uses: actions/setup-node@v1
         with:
-          node-version: 16.13
+          node-version: 16.15
 
       - uses: actions/cache@v2
-        id: deps-cache
+        id: mix-cache
         with:
-          path: deps
-          key: ${{ runner.os }}-deps-${{ hashFiles(format('{0}/mix.lock', github.workspace)) }}
-          restore-keys: |
-            ${{ runner.os }}-deps-
-
-      - uses: actions/cache@v2
-        id: build-cache
-        with:
-          path: _build
-          key: ${{ runner.os }}-build-${{ matrix.otp-version }}-${{ matrix.elixir-version }}-${{ hashFiles(format('{0}/mix.lock', github.workspace)) }}
+          path: |
+            deps
+            _build
+            priv/plts
+          key: ${{ runner.os }}-mix-${{ matrix.otp-version }}-${{ matrix.elixir-version }}-${{ hashFiles(format('{0}/mix.lock', github.workspace)) }}
```

Comment:
Since I added the _Persistent Lookup Table_ folder to optimize the Dialyzer workload, I merged the `mix`-related caching steps into a single hash key so everything related to `mix` is cached altogether.